### ISSUE TITLE
Does this have any sort of roadmap?

### ISSUE THREAD

**Issue by chandlervdw** [arnaudlewis/react-native-kit#1]
As it stands, working with the javascript-kit, `prismic.io`, is kind of a pain. It'd be awesome to have more of a plug-n-play solution.

**Comment by arnaudlewis** [comment]
Hi, I'm well aware of the few problems with the official javascript-kit, i have a whole new alternative on my github https://github.com/arnaudlewis/prismic-jskit-v4 available on npm alw-prismic.io.
For the react native fork, it's just a fast adaptation of the official kit to make it work.
i'll work on a unify version for JS and react native with the new approach i tried on js-kit v4

**Comment by chandlervdw** [comment]
@arnaudlewis gracias! Would you mind pinging here when/if you ever get react-native baked into `alw-prismic.io`?

**Comment by arnaudlewis** [comment]
yeah no problem, i'll do this ;) But in the future, it'll be necessary to fork my js-kit to have something very specific to react native and generate native components for structuredText of complex fragments like this. I'll keep you in touch.

**Comment by arnaudlewis** [comment]
Hi @chandlervdw, I've been working on a new version of the JavaScript kit these days, i will release it soon but I'll also release a react native kit version a short time after based on this new JS kit. I'll keep you in touch if you wanna try.

**Comment by chandlervdw** [comment]
Thanks! That'd be awesome.

--
Chandler
